application/classes/Ushahidi/Repository/Config.php

```diff
@@ -61,6 +61,7 @@ public function update(Entity $entity)
 	public function groups()
 	{
 		return [
+			'features',
```

Comment:
So the problem is this enables writing as well as reading by an admin.. I'll have a think about the best solution